src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -0,0 +1,623 @@
+/***************************************************************************
+                           dlgprefcontrols.cpp  -  description
```

Comment:
dlgprefdeck.cpp ?

Reply:
Good catch, thanks. That comment could be removed entirely.